app/src/main/java/agh/ics/oop/AbstractWorldMap.java

```diff
@@ -13,16 +14,22 @@ public void positionChanged(Vector2d oldPosition, Vector2d newPosition) {
         Animal animal = animals.get(oldPosition);
         animals.remove(oldPosition);
         animals.put(newPosition, animal);
+        mapBoundary.positionChanged(oldPosition, newPosition);
```

Comment:
można by zainstalować to jako osobnego obserwatora przy umieszczaniu zwierzątka na mapie